migrations/000011_alter_workloads_table_workloadtype.up.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TYPE workloadtype RENAME TO workloadtype_old;
+
+CREATE TYPE workloadtype AS ENUM ('daemonset', 'deployment', 'deploymentconfig', 'replicaset', 'replicationcontroller', 'statefulset');
```

Comment:
@saltgen Any custom ``TYPE`` enum value can't be deleted but only be appended. We will have to create new ``TYPE`` with proper ordering, rename previous ``TYPE `` to something different name then link new created TYPE to referenced tables ie ``workloads`` table in this case.

Reply:
Ditto